pkg/crc/machine/start.go

```diff
@@ -133,6 +133,10 @@ func (client *client) Start(ctx context.Context, startConfig StartConfig) (*Star
 			return nil, errors.Wrap(err, "Error getting bundle metadata")
 		}
 
+		if err := bundle.Use(filepath.Base(startConfig.BundlePath)); err != nil {
+			return nil, err
+		}
+
```

Comment:
This could be moved to `getCrcBundleInfo()` which is only used here, or maybe `bundle.Use()` could be called a single time around the `// Post-VM start` comment? At this point we don't have ssh connectivity to the VM yet, so I don't think the code tries to use oc before that.

Reply:
`getCrcBundleInfo()` is only used at #131 and we should use `bundle.Use` there instead `bundle.Get` in the `getCrcBundleInfo()` and `bundle.Use` should also return the `bundleInfo` as mentioned https://github.com/code-ready/crc/pull/2062#discussion_r588125500 here.